### DISCUSSION TITLE
GitHub Mobile iOS - read a file issue.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#42631] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.93.0 (85890385)
Device: iPhone 11 Pro Max (iOS 16.2)
</details>

Using the iOS app, I was reading a read me, but the app cap off the full file. I couldn't finish reading it since it seems the have an issue on how much text can be viewed/loaded. The publisher had a table, clicking on that took me to a blank space, tapping in the screen took me to end of loaded file. Hope that makes sense. Thank you.

**Comment by Allofthes** [comment]
> ℹ️ Additional Debug Info
> Using the iOS app, I was reading a read me, but the app cap off the full file. I couldn't finish reading it since it seems the have an issue on how much text can be viewed/loaded. The publisher had a table, clicking on that took me to a blank space, tapping in the screen took me to end of loaded file. Hope that makes sense. Thank you.
